pkg/certifier/osv/osv.go

```diff
@@ -102,8 +103,9 @@ func generateDocument(packNodes []*root_package.PackageNode, vulns []osv_scanner
 			Type:   processor.DocumentITE6Vul,
 			Format: processor.FormatJSON,
 			SourceInformation: processor.SourceInformation{
-				Collector: INVOC_URI,
-				Source:    INVOC_URI,
+				Collector:   INVOC_URI,
+				Source:      INVOC_URI,
```

Comment:
not part of this PR initially, but would be useful (like scorecard) to have this be "osv_certifier" or something similar

Reply:
so i had that but removed it lol.  I will add it back in